src/components/ListItem.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
```

Comment:
You should really not have unidentified `div`s in your code. You should give them a class so we can understand this better. Your future self will thank you! Refrain from using IDs anywhere in a Vue component, so you should assign a class to it. I like to use the pattern `<componentName>-view` for view, so this one would have the class, `list-view`, but you can use whatever name you like. According to the Vue style guides, you should use the appropriate naming convention for CSS, with hyphens instead of camel-case.

Reply:
I used list-item-view.